### DISCUSSION TITLE
Feedback from Maria

### DISCUSSION THREAD

**Discussion by Maria-Valera** [mdbootstrap/TW-Elements#1699] [Kind words & general feedback]
Hello Tailwind Elements team!

I just started using Tailwind Elements and I wanted to give some feedback.
Overall, I think it's a fantastic product and it's really helped me *speed up my web development process*.

However, I have ideas of what could be improved. Here are my suggestions: 
- 
- 
- 

Shout out to [Michal](https://twitter.com/ascensus_mdb) & [Dawid](https://twitter.com/dawidadach) for starting this & great thanks to @Trochonovitz @juujisai @Martussky @smolenski-mikolaj for all the work put in the latest release!

Keep coding!

**Comment by filipkappa** [comment]
I think you forgot to add your suggestions ;) but still - thanks for your support!
